### PR TITLE
Cypress clean installation on the project

### PR DESCRIPTION
**Pull request by reskin7**
## Summary
- Installed cypress

## Screenshot
image

## Trello Card
<https://wolox-support.atlassian.net/browse/QAT-3245>

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (gabrielbori) [inline cypress.json]: Do not push empty files

File: cypress/fixtures/example.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (gabrielbori) [inline cypress/fixtures/example.json]: delete example file

File: cypress/support/commands.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
```
Comment (gabrielbori) [inline cypress/support/commands.js:3]: Do not push file with only comments

**Review by gabrielbori — CHANGES_REQUESTED** [review]

File: cypress/support/commands.js (hunk 3 above)
Comment (reskin7) [reply cypress/support/commands.js:3]: image
Those comments help a lot, they explain how commands.js works on cypress

Comment (reskin7) [reply cypress/support/commands.js:3]: I will need them later

**Review by gabrielbori — APPROVED** [review]
Nice

**Pull request merged**